tracekit.js

```diff
@@ -1225,8 +1225,13 @@ if (!TraceKit.linesOfContext || TraceKit.linesOfContext < 1) {
 }
 
 
-
-// Export to global object
-window.TraceKit = TraceKit;
+// UMD export
+if (typeof module != 'undefined' && module.exports && this.module !== module) {
+    module.exports = TraceKit;
+} else if (typeof define === 'function' && define.amd) {
+    define(TraceKit);
+} else {
+    window.TraceKit = TraceKit;
+}
 
 }(typeof window !== 'undefined' ? window : global));
```

Comment:
Yeah I agree the top would be the best. I'll take a look into this and do lots of testing and merge it in. It's a great change just need to be sure everything still works as expected.